setup.py

```diff
@@ -13,17 +13,17 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 """
-from setuptools import setup
+from setuptools import setup, find_packages
 
 setup(
     name='Cibyl',
     author='RHOS CRE Team',
     license='Apache',
     description='Cibyl is tool that models ci system by'
     'command-line interface for querying CI environments',
-    version='0.0.1',
+    version='0.0.2',
     url='https://github.com/rhos-infra/cibyl',
-    packages=['cibyl'],
+    packages=find_packages('cibyl'),
```

Comment:
This should be just `.` according to https://github.com/pypa/setuptools/issues/1571#issuecomment-435604162